src/DynamoMLDataPipeline/DynamoMLDataPipeline.cs

```diff
@@ -78,6 +78,8 @@ public static string ConstructCreateAssetRequestBody(string schemaNamespaceId, s
             parameterComponent.AddParameterFromSchema("<Add dynamo host>", hostSchema);
             parameterComponent.AddParameterFromSchema("<Add dynamo version>", dynamoVersionSchema);
 
+            Analytics.TrackEvent(Actions.DynamoMLDataPipieline, Categories.DynamoMLDataPipelineOperations, "Host", Convert.ToInt32(hostSchema));
```

Comment:
Isn't host name already part of each data entry? Do we still need to send it separately?

Reply:
Removed this.